### PR TITLE
Eval loader transforms

### PR DESCRIPTION
**Pull request by zcain117**
Verified that transformed images during eval loop match those of Torchvision's train/eval script:
https://github.com/pytorch/examples/blob/master/imagenet/main.py#L219

Verified that inception_v3 does not crash during training or eval in our version like it does in Torchvision's example script

### CODE REVIEW

**Comment by taylanbil** [comment]
> Does anyone know if the max() call would be triggered every time we transform an image? If so, I can call this outside of the transform and just use a value on the inside

I don't think you have to do that.

**Comment by taylanbil** [comment]
Do we need this change? Is the purpose having the same validation step as in torchvision?

**Comment by zcain117** [comment]
> Do we need this change? Is the purpose having the same validation step as in torchvision?

Yes exactly. We want to verify we match the accuracy from this page: https://pytorch.org/docs/stable/torchvision/models.html

If we use randomized transforms during our eval loop, we'll never know if our accuracy is different due to the randomness or due to our model weights. I'd also be surprised if we could ever approach those published accuracies with random cropping / flipping since it makes the task harder

**Comment by jysohn23** [comment]
> Do we need this change? Is the purpose having the same validation step as in torchvision?

I think in general when running eval we don't want any augmentation, right?
